src/commands/deployWorkspaceProject/getDefaultValues/getDefaultContainerAppsResources/getResourceHelpers.ts

```diff
@@ -16,6 +16,7 @@ export async function getResourcesFromContainerAppHelper(context: ISubscriptionA
 
     const managedEnvironments: ManagedEnvironment[] = await uiUtils.listAllIterator(client.managedEnvironments.listBySubscription());
     const managedEnvironment = managedEnvironments.find(env => env.id === containerApp.managedEnvironmentId);
+    context.telemetry.properties.managedEnvironmentCount = String(managedEnvironments.length);
```

Comment:
Since this is only going to be called for `deployWorkspaceProject`, I'd recommend putting this somewhere else since I imagine that we want it for every command involving a container app.

You may be able to make a helper command to get all the azure resource count telemetry that you want. In the past, it was easy enough to just grab a parent's children and see how many there were, but since that's not possible anymore here are a couple of ways I can think of:

1. Save a global context that updates every time the listAll call is made. Would probably want to make a wrapper so that it's consistently tracked/updated.
2. Just make a list call of what you want count of wherever else we want the telemetry.

Honestly though, I'm not sure _how_ valuable it is to know managed environment count, especially since way 1 is kind of a pain and way 2 slows down the command. 

I see it's value mostly in 3 places:
1. `deployWorkspaceProject` (here)
2. `createManagedEnviornment`
3. `createContainerApp`

Reply:
It's not super important, I only logged it because we were already making a query and thought it could be interesting.  I don't really see a deep need to log it for any specific purposes though